receipt_parser/importer.py

```diff
@@ -76,14 +76,22 @@ def rotate_image(input_file, output_file, angle=90):
     :return: void
         Rotates image and saves result
     """
+
+    if (angle == 0):
```

Comment:
We should move those conditions out for separation of concerns:
the function should just rotate the image unconditionally; the logic whether to rotate the image should be outside.
Furthermore I have mixed feelings about reverting the sign of the rotation. What was the use-case for that?

So basically

```python
if angle != 0:
  rotate_image(input_file, output_file, angle)
```

What do you think? 😃

Reply:
As @monolidth said, this should rotate the image in the correct possition. Maybe it's not trivial.